test/framework/src/main/java/org/opensearch/test/OpenSearchSingleNodeTestCase.java

```diff
@@ -211,7 +212,10 @@ protected final Collection<Class<? extends Plugin>> pluginList(Class<? extends P
 
     /** Additional settings to add when creating the node. Also allows overriding the default settings. */
     protected Settings nodeSettings() {
-        return Settings.EMPTY;
+        // By default, for tests we will put the target slice count of 2. This will increase the probability of having multiple slices
+        // when tests are run with concurrent segment search enabled. When concurrent segment search is disabled then it's a no-op as
+        // slices are not used
+        return Settings.builder().put(SearchBootstrapSettings.CONCURRENT_SEGMENT_SEARCH_TARGET_MAX_SLICE_COUNT_KEY, 2).build();
```

Comment:
Maybe move this to `newNode()` where the other defaults are defined and leave this implementation as returning `EMPTY`?